kolibri/plugins/learn/assets/src/views/LibraryPage/AccessibleFolderCard.vue

```diff
@@ -0,0 +1,138 @@
+<template>
+
+  
+  <KCard
+    to="to"
+    :headingLevel="headingLevel"
+    layout="horizontal"
+    thumbnailDisplay="large"
+    :title="contentNode.title"
+    :thumbnailSrc="thumbnailSrc"
+    :thumbnailScaleType="thumbnailScaleType"
+    :thumbnailAlign="thumbnailAlign"
+    :style="{ height: '172px', margin: '16px 0 16px 0' }"
+  >
+    <template #thumbnailPlaceholder>
+      <div class="default-folder-icon" >
+        <KIcon
+          icon="topic"
+          :color="$themePalette.grey.v_600"
+          class=""
+        />
+      </div>
+    </template>
+
+    <template #belowTitle>
+      <div
+        class="header-bar"
+        :style="headerStyles "
+      >
+        <KIcon
+          icon="topic"
+          :color="$themePalette.grey.v_800"
+          class="folder-header-bar"
+        />
+        <p
+          class="folder-header-text"
+          :style="{ color: $themePalette.grey.v_600 }"
+        >
+          {{ coreString('folder') }}
+        </p>
+      </div>
+    </template>
+
+  </KCard>
+
+</template>
+
+
+<script>
+
+  import { validateLinkObject } from 'kolibri.utils.validators';
+  import commonCoreStrings from 'kolibri.coreVue.mixins.commonCoreStrings';
+  import commonLearnStrings from '../commonLearnStrings';
+
+  export default {
+    name: 'AccessibleFolderCard',
+    mixins: [commonLearnStrings, commonCoreStrings],
+    props: {
+      to: {
+        type: Object,
+        required: true,
+        validator: validateLinkObject,
+      },
+      contentNode: {
+        type: Object,
+        required: true,
+      },
+      headingLevel: {
+        type: Number,
+        required:true
+      },
+      thumbnailSrc: {
+        type: String,
+        default: null,
+      },
+      thumbnailScaleType: {
+        type: String,
+        default: "centerInside",
+      },
+      thumbnailAlign: {
+        type: String,
+        default: "right",
+      },
+    },
+    computed: {
+      headerStyles() {
+        let styles = {};
+        if (!this.contentNode.is_leaf) {
```

Comment:
Let's remove `is_leaf` conditions from the whole component since it is meant to work specifically with folders (that aren't leaves) so we don't really need to be making any decisions on whether it's a resource or folder inside this component. This will be done outside it when we're deciding whether resource or folder card should be displayed.